.github/workflows/unit-tests.yml

```diff
@@ -20,5 +20,8 @@ jobs:
           java-version: '17'
           distribution: 'temurin'
 
+      - name: Set env
```

Comment:
Or can we remove that and test `GITHUB_JOB = "test"` or even just that `GITHUB_JOB` exists? There are [plenty of environment variables](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables) set by the GitHub runners.